docs/design/proposals/encryption-with-vault-tokens.md

```diff
@@ -243,9 +243,9 @@ kind: secret
 metadata:
   name: vault-certificates
 stringData:
-  cert: |
+  cert1: |
```

Comment:
Can there be multiple certificates in the Secret? Or should this Secret contain only a `cert` and `key`?

Reply:
definitely more than one key:value can be part of the secret. So, its fine.

Reply:
regarding more than one cert will be respected by code, I have to check.  @Madhu-1 22ae4a0b16acaa16642dff09101f38d7cceb2a20 looks like changed this key names from `ca.cert` and `tls.cert` to `cert` and `cert` .  Is this valid configuration ?

Reply:
I'd like to know the answer of this, before approving the change.

Reply:
the secret data part is a map[string]string we cannot have duplication keys, I think we need to remove adding multiple certs to the same secret.

Reply:
the key should always remain const `cert` we cannot have cert1 and cert2 here.

Reply:
Yeah, we should have uniqueness in key and only one key is respected in code, so adjusted the same. @nixpanic @Madhu-1 ptal